package-system/OpenMesh/FindOpenMesh.cmake.Linux

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(LIB_NAME "OpenMesh")
+
+set(TARGET_WITH_NAMESPACE "3rdParty::$${LIB_NAME}")
+if (TARGET $${TARGET_WITH_NAMESPACE})
+    return()
+endif()
+
+set($${LIB_NAME}_INCLUDE_DIR $${CMAKE_CURRENT_LIST_DIR}/$${LIB_NAME}/include)
+set($${LIB_NAME}_LIBS_DIR $${CMAKE_CURRENT_LIST_DIR}/$${LIB_NAME}/lib)
+
+set($${LIB_NAME}_LIBRARY_DEBUG   $${$${LIB_NAME}_LIBS_DIR}/libOpenMeshCored.a)
+set($${LIB_NAME}_LIBRARY_RELEASE $${$${LIB_NAME}_LIBS_DIR}/libOpenMeshCore.a)
+
+set($${LIB_NAME}_LIBRARY
+    "$$<$$<CONFIG:profile>:$${$${LIB_NAME}_LIBRARY_RELEASE}>"
+    "$$<$$<CONFIG:release>:$${$${LIB_NAME}_LIBRARY_RELEASE}>"
+    "$$<$$<CONFIG:debug>:$${$${LIB_NAME}_LIBRARY_DEBUG}>")
+
+add_library($${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
```

Comment:
this is a static library, not an interface, so declare it as STATIC IMPORTED GLOBAL, instead of INTERFACE IMPORTED GLOBAL  (same with the rest of the find fiiles).  And supply an IMPORTED_LOCATION, not target_link_libraries
```
set_target_properties(${TARGET_WITH_NAMESPACE} PROPERTIES IMPORTED_LOCATION "${${LIB_NAME}_LIBRARY}")
```